openwisp_monitoring/device/base/models.py

```diff
@@ -93,6 +86,19 @@ def data_user_friendly(self):
         # reformat expiry in dhcp leases
         for lease in data.get('dhcp_leases', []):
             lease['expiry'] = datetime.fromtimestamp(lease['expiry'], tz=tz('UTC'))
+        if 'resources' in data and 'disk' in data['resources']:
+            # convert kibibytes to bytes
+            KiB = 1024
```

Comment:
Since the metric is called used_bytes IMO the conversion should be done in https://github.com/openwisp/openwisp-monitoring/pull/76/ to make sure that we read bytes from influx

Reply:
The above change is also not changing the way how the data is stored in influxdb but just changing the way it's to be displayed. I think, I shall remove the above one and rather change the lua script to pass data in bytes, right now there seems a mismatch between the name and the actual data (this is incorrect)

Reply:
opened https://github.com/openwisp/lua-monitoring/pull/16 to correct this.

Reply:
Merged